pkg/wkspace/minimal.go

```diff
@@ -295,3 +295,21 @@ func templateTerraformInputs(name, vals string) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
+
+func getRepoPath(name string) (string, error) {
+	wd, err := os.Getwd()
```

Comment:
Would it be better to use the ProjectRoot helper to just find the appropriate root then generate the path from there?   This logic seems brittle

Reply:
thanks, done